docs/workflow/testing/libraries/filtering-tests.md

```diff
@@ -1,8 +1,18 @@
 # Filtering libraries tests using traits
 
-The tests can be filtered based on xunit trait attributes defined in [`Microsoft.DotNet.XUnitExtensions`](https://github.com/dotnet/arcade/tree/master/src/Microsoft.DotNet.XUnitExtensions). These attributes are specified above the test method's definition. The available attributes are:
+The tests can be filtered based on xunit trait attributes defined in [`Microsoft.DotNet.XUnitExtensions`](https://github.com/dotnet/arcade/tree/master/src/Microsoft.DotNet.XUnitExtensions).
 
-#### OuterLoopAttribute
+Some of the attributes take arguments to restrict filtering to a subset of configurations, including platform, runtime, and target framework moniker:
+
+`TestPlatforms` is defined [here](https://github.com/dotnet/arcade/blob/master/src/Microsoft.DotNet.XUnitExtensions/src/TestPlatforms.cs).
+
+`TestRuntimes` (CoreCLR, Mono) is defined [here](https://github.com/dotnet/arcade/blob/main/src/Microsoft.DotNet.XUnitExtensions/src/TestRuntimes.cs).
+
+`TargetFrameworkMonikers` (Netcoreapp, NetFramework) is defined [here](https://github.com/dotnet/arcade/blob/main/src/Microsoft.DotNet.XUnitExtensions/src/TargetFrameworkMonikers.cs).
+
+These attributes are specified above the test method's definition. The available attributes are:
+
+## OuterLoopAttribute
 
 ```cs
```

Comment:
another outerloop reason is that the test is disruptive, eg., it pops up a dialog.

we have been de-facto using outerloop for tests that are a bit less reliable, although that practice is dubious now that we are driving outerloop clean.